source/_integrations/teslemetry.markdown

```diff
@@ -190,3 +190,7 @@ These are the entities available in the Teslemetry integration. Not all entities
 |Sensor|Power|Yes|
 |Sensor|State|Yes|
 |Sensor|Vehicle|Yes|
+
+## Vehicle sleep
+
+Constant API polling will prevent most Model S and Model X vehicles manufactured before 2021 from sleeping, so the Teslemetry integration will stop polling these vehicles for 15 minutes, after 15 minutes of inactivity. You can call the `update_entity` service at any time to force polling the API, which will reset the timer.
```

Comment:
If someone starts charging or in other way also operates the vehicle it would also reset the timer right?

Reply:
Correct, should I elaborate on what inactivity means? Charging, sitting in the vehicle, or having sentry mode on is what I currently use.

Reply:
Maybe something more like "When the car is sleeping/not operating ,you can use the `homeassistant.update_entity` service to wake it up at any given time".
But I'll leave the final review to the people much better at wordings than I am. :)